es-model-archive/kf-es-model-v0.5.0_1.2.0/file_centric.mapping.json

```diff
@@ -0,0 +1,528 @@
+                }, 
+                "shipment_origin": {
+                  "type": "keyword"
+                }, 
+                "source_text_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "source_text_tissue_type": {
+                  "type": "keyword"
+                }, 
+                "source_text_tumor_descriptor": {
+                  "type": "keyword"
+                }, 
+                "spatial_descriptor": {
+                  "type": "keyword"
+                }, 
+                "uberon_id_anatomical_site": {
+                  "type": "keyword"
+                }, 
+                "volume_ml": {
+                  "type": "float"
+                }
+              }, 
+              "type": "nested"
+            }, 
+            "consent_type": {
+              "type": "keyword"
+            }, 
+            "diagnoses": {
+              "properties": {
+                "age_at_event_days": {
+                  "type": "integer"
+                }, 
+                "diagnosis": {
+                  "type": "keyword"
+                }, 
+                "diagnosis_category": {
+                  "type": "keyword"
+                }, 
+                "external_id": {
+                  "type": "keyword"
+                }, 
+                "icd_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "kf_id": {
+                  "type": "keyword"
+                }, 
+                "mondo_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "ncit_id_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "source_text_diagnosis": {
+                  "type": "keyword"
+                }, 
+                "source_text_tumor_location": {
+                  "type": "keyword"
+                }, 
+                "spatial_descriptor": {
+                  "type": "keyword"
+                }, 
+                "uberon_id_tumor_location": {
+                  "type": "keyword"
+                }
+              }, 
+              "type": "nested"
+            }, 
+            "diagnosis_category": {
+              "type": "keyword"
+            }, 
+            "ethnicity": {
+              "type": "keyword"
+            }, 
+            "external_id": {
+              "type": "keyword"
+            }, 
+            "family": {
+              "properties": {
+                "available_data_types": {
+                  "type": "keyword"
+                }, 
+                "external_id": {
+                  "type": "keyword"
+                }, 
+                "family_members": {
+                  "properties": {
+                    "affected_status": {
+                      "type": "boolean"
+                    }, 
+                    "alias_group": {
+                      "type": "keyword"
+                    }, 
+                    "available_data_types": {
```

Comment:
similar question as a previous one, `available_data_types` for a family member is not from the participant entity of the `dataservice`, keep the field only when ETL will populate the value of it.